contrib/cirrus/runner.sh

```diff
@@ -55,31 +60,66 @@ function _run_unit() {
 }
 
 function _run_apiv2() {
+    if _only_changes_are_in docs hack test/{buildah-bud,e2e,system,upgrade}
+    then
+        return
+    fi
+
     source .venv/requests/bin/activate
     make localapiv2 |& logformatter
 }
 
 function _run_compose() {
+    if _only_changes_are_in docs hack test/{apiv2,buildah-bud,e2e,system,upgrade}
+    then
+        return
+    fi
+
     ./test/compose/test-compose |& logformatter
 }
 
 function _run_compose_v2() {
+    if _only_changes_are_in docs hack test/{apiv2,buildah-bud,e2e,system,upgrade}
+    then
+        return
+    fi
+
     ./test/compose/test-compose |& logformatter
 }
 
 function _run_int() {
+    if _only_changes_are_in docs hack test/{apiv2,buildah-bud,system,upgrade}
+    then
+        return
+    fi
+
```

Comment:
There's a sneaky corner-case gotcha here: The `localintegration` target depends on both `test-binaries` and `ginkgo` which both reference `$(BUILDTAGS)`.  Maybe you see where I'm going with this...`$(BUILDTAGS)` is defined by calling a whole bunch of `hack/` scripts :cry:

Maybe this is okay?  I think the most likely consequence is someone breaks something like `hack/btrfs_tag.sh` and it merges.  Literally everybody will immediately notice as the PR universe crumbles.

I'm not sure there's a sane way to deal with this, other than removing `hack` here (and maybe elsewhere).

I'm choosing to stick my head in the sand regarding problems keeping Makefile changes in-sync with `_only_changes_are_in docs` conditionals.  I think we can rely on the Makefile being so damn scarry nobody will touch it :grin: